packages/design-system-tokens/src/themes/core-components.ts

```diff
@@ -418,6 +418,16 @@ export const components: AnyTokenValues = {
     '-trigger__color':                            t.color['primary'],
   },
 
+  'typography': {
+    '-heading__font-family':                      'inherit',
+    '-heading__font-weight':                      t.font['weight-bold'],
+    '-heading-4xl__font-weight':                  'inherit',
+    '-heading-4xl__line-height':                  'inherit',
+    '-text__color':                               'inherit',
+    '-text__font-family':                         'inherit',
```

Comment:
Wondering about naming here and using something like   
  
\- text-heading for headings and text-body for body text?

Reply:
Adding `-body` here as I think removing the secondary name makes it too generic, gotta leave room for more specificity in the future.